### ISSUE TITLE
Add Silent Mode for JobQueue

### ISSUE THREAD

**Issue by accraze** [discogs/pymongo-job-queue#8]
When you use the job queue, it prints `waiting!` every 5 seconds if there are no jobs in the queue.
https://github.com/discogs/pymongo-job-queue/blob/master/pymjq/jobqueue.py#L95

It would be awesome to have a `silent` kwarg on JobQueue initialization so we don't see the `waiting!` message clogging up the terminal in development.

Maybe something like:

``` python
from pymongo import MongoClient
from pymjq import JobQueue
client = MongoClient("localhost", 27017)
db = client.job_queue
jobqueue = JobQueue(db, silent=True)
```

**Comment by enriquegh** [comment]
I'll give this a shot

**Comment by accraze** [comment]
go for it @enriquegh, let me know if you need any help!

**Comment by shanmbic** [comment]
Hi Guys, I added the 'silent' keyword , which is set to False as default . While waiting for new jobs, if it is set to True it will not print 'waiting'. Am I missing something here ?? Pull request : https://github.com/discogs/pymongo-job-queue/pull/9

**Comment by accraze** [comment]
Looks good @shanmbic thanks again! 

Closed by #9
